jobs/sql/extractions/backup_restore_dates.sql

```diff
@@ -0,0 +1,15 @@
+drop temporary table if exists temp_backup_restore_dates;
+create temporary table temp_backup_restore_dates
+(datetime_type varchar(255),
+datetime       text);
+
+insert into temp_backup_restore_dates
+select 'backup',global_property_value('percona_backup_date', null);
+
+insert into temp_backup_restore_dates
+select 'restore',global_property_value('percona_restore_date', null);
+
```

Comment:
You could probably get a bit more fancy and do away with the temp table, and just have a single sql query here,  using the substring_index function to retrieve the "type" based on the gp name, and using the like function to restrict to the global properties in question, but this is fine and probably easier to read and more explicit.